core/src/main/scala/io/gearpump/cluster/ClusterConfig.scala

```diff
@@ -142,7 +142,7 @@ object ClusterConfig {
 
     if (!akka.util.Helpers.isWindows) {
 
-      // Change the akka.scheduler.tick-duration to 10ms for Linux or Mac
+      // Change the akka.scheduler.tick-duration to 1 ms for Linux or Mac
```

Comment:
I think it's better to load window specific or linux specific configurations here than later to decide whether a config belongs to windows or linux